.github/workflows/lint.yml

```diff
@@ -454,6 +454,34 @@ jobs:
             false
           fi
 
+  actionslint:
```

Comment:
I think we can also have the check inside shell check so it's one liner to download actionlint, instead of using `go install` which also compiles the project.

Example (put this after the shellcheck is installed):
```
       - name: Actionlint GitHub action yml files
        shell: bash
        run: |
          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
          ./actionlint -ignore '"needs" section should not be empty'
          rm ./actionlint
```

Reply:
oh, I mean here to reuse the existing shellcheck, and also make sure run `./actionlint` as it's not in PATH

https://github.com/pytorch/pytorch/blob/master/.github/workflows/lint.yml#L219

Reply:
Ah makes sense, it's nicer that way too since it doesn't add another job entry to the signals box on GitHub